README.md

```diff
@@ -14,10 +14,11 @@ Memcached protocol 'wrapper' for Tarantool.
 
  * Tarantol 1.6.8+ with header files (tarantool && tarantool-dev packages).
  * Cyrus SASL library (with header files)
- * Python >= 2.7, <3 with next packages (for testing only):
+ * Python >= 2.7 with next packages (for testing only):
```

Comment:
Maybe Python >= 3 is better because we are testing with Python3?

Reply:
To be honestly, we need to specify here all supported Python versions. However, 2.x is EOL, and we can limit versions by 3.x. So I replaced to ">= 3.7" (Python 3.7 is a minimal supported version, see https://endoflife.date/python):

```diff
--- a/README.md
+++ b/README.md
@@ -14,7 +14,7 @@ Memcached protocol 'wrapper' for Tarantool.
 
  * Tarantol 1.6.8+ with header files (tarantool && tarantool-dev packages).
  * Cyrus SASL library (with header files)
- * Python >= 2.7 with next packages (for testing only):
+ * Python >= 3.7 with next packages (for testing only):
    - PyYAML
    - msgpack-python
    - six==1.9.0
```